ajax/libs/ityped/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "author": "Luis Vinicius <luisviniciusbarreto@gmail.com>",
+  "description": "Dead simple animated typing, with no dependencies.",
+  "filename": "index.min.js",
+  "homepage": "https://ityped.surge.sh/",
+  "keywords": [
+    "typed",
+    "ityped",
+    "letters",
+    "animation",
+    "typing",
+    "animated typing"
+  ],
+  "name": "ityped",
```

Comment:
Please put `name` in the first line if possible to have people see it in the first glance.